app/src/shared/components/Modal/index.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React, { type Node } from 'react'
+import { createPortal } from 'react-dom'
+
+import BodyClass, { NO_SCROLL } from '$shared/components/BodyClass'
+import Context from '$shared/contexts/Modal'
+import NoModalRootError from '$shared/errors/NoModalRootError'
+
+type Props = {
+    children: Node,
+}
+
+class Modal extends React.Component<Props> {
+    static contextType = Context
+
+    static isOpen() {
+        return BodyClass.includes(NO_SCROLL)
```

Comment:
Maybe add an explicit class like `MODAL_OPEN`? Also, regarding SSR, these classes won't be available immediately with current implementation of `BodyClass`.

Reply:
Yeah, luckily we're not gonna use it for this. 🤠